SCons/Script/SConsOptions.py

```diff
@@ -138,7 +140,8 @@ def __getattr__(self, attr):
         'stack_size',
         'warn',
         'silent',
-        'no_progress'
+        'no_progress',
+        'experimental'
```

Comment:
if you end this with a comma it won't flag as a changed line next option we add...

Reply:
k Updated.